README.md

```diff
@@ -1,36 +1,39 @@
 # mongoose-harakiri
 
-**Mongoose Harakiri ensures a stable connection with mongoose, and kills itself if it can't establish one.**
+**Mongoose Harakiri ensures a stable connection with mongoose, and kills itself if it can't establish one or looses it.**
+
+## Motivation
+
+In the era of cloud computing, orchestration tools, and Kubernetes, we need reliable database connections more than ever. Since the orchestration tool can respawn a container that exits with an error, it's preferable to surface connection problems and explicitly try to re-initiate the connection by killing and spawning the whole container again. Also, applications that rely on databases shouldn't be alive if they aren't actually connected to a database, because they are in fact useless.
 
 ## Behavior
+
 - Replaces good old `mongoose.connect` call
 - Doesn't create a new connection if mongoose is already connected
 - If it can't connect, it kills itself, so no more hanging processes!
-- If it can't re-connect, it kills itself, so no more hanging processes!
+- If it losses connection, it kills itself. This is useful in sharded clusters where reconnect to Mongos is [broken in MongoClient](https://jira.mongodb.org/browse/NODE-1340)
```

Comment:
`loses` instead of `losses`. please keep `so no more hanging processes!`, it's a pattern.